src/jit/codegencommon.cpp

```diff
@@ -3240,7 +3240,11 @@ void CodeGen::genGenerateCode(void** codePtr, ULONG* nativeSizeOfCode)
     }
 #endif // EMIT_TRACK_STACK_DEPTH
 
-    *nativeSizeOfCode                 = codeSize;
+    if (nativeSizeOfCode != nullptr)
+    {
+        *nativeSizeOfCode = codeSize;
+    }
+
```

Comment:
I'm a little surprised this would be needed. If you found a code path that calls here with nativeSizeOfCode = NULL we should probably fix the caller. But if you let me know what the callstack is that gets here I can help assess.